querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx

```diff
@@ -702,7 +704,10 @@ class DataDocQueryCellComponent extends React.PureComponent<IProps, IState> {
                 };
             }
         });
-        this.setState({ samplingTables });
+        this.setState({
```

Comment:
do you recall why this function is async but there is no await?

Reply:
probably it was implemented by getting the tables info in this function and somehow I changed it and forgot to remove the async :)